static/templates/settings/admin_user_list.hbs

```diff
@@ -1,7 +1,7 @@
 <tr class="user_row{{#unless is_active}} deactivated_user{{/unless}}" data-user-id="{{user_id}}">
     <td>
         <span class="user_name" >{{full_name}} {{#if is_current_user}}<span class="my_user_status">{{t '(you)' }}</span>{{/if}}</span>
-        <i class="fa fa-ban deactivated-user-icon" title="{{t 'User is deactivated' }}" {{#if is_active}}style="display: none;"{{/if}}></i>
+        <i class="fa fa-ban deactivated-user-icon" data-tippy-content="{{full_name}}" {{#if is_active}}style="display: none;"{{/if}}></i>
```

Comment:
Why have you changed the tooltip to have full name? Name is already shown besides the icon and am not seeing anything about this in the Issue too.

Reply:
Sorry, i read the issue description wrong.